### PR TITLE
Update svgr config to include role=img

### PR DESCRIPTION
**Pull request by Rhotimee**
#### Summary
`role="img"` is no longer present by default. Instead, it can be introduced as an optional config. 
This PR is to update the `svgr.config.js` to include `role="img"`.

https://github.com/gregberge/svgr/compare/v6.5.0...v6.5.1

### CODE REVIEW

**Comment by CarlosCortizasCT** [comment]
When generating the icons in this PR all of them have small changes regarding removing break lines.
If this is caused from the new version, I think we should include those changes in this PR.

**Comment by Rhotimee** [comment]
> When generating the icons in this PR all of them have small changes regarding removing break lines. If this is caused from the new version, I think we should include those changes in this PR.

Yes, you're right. I didn't include it here because that was already in the Tag PR and I didn't want to deal with the conflict.
Since that has been reverted, I have included it [3b83d5c](https://github.com/commercetools/ui-kit/pull/2386/commits/3b83d5c78ad199e8d142492bf9a69c01550778f7)

File: svgr.config.js [hunk 1]
```diff
@@ -92,6 +92,8 @@ module.exports = {
       'prefixIds',
     ],
   },
+  svgProps: { role: 'img' },
+  typescript: true,
```
Comment (emmenko) [inline svgr.config.js:96]: 💯

**Review by emmenko — APPROVED** [review]
Thanks!

**Review by CarlosCortizasCT — APPROVED** [review]
👍

**Pull request merged**